src/ert/analysis/_es_update.py

```diff
@@ -175,11 +175,17 @@ def _load_observations_and_responses(
         # Exit early if some observations are pointing to non-existing responses
         raise ErtAnalysisError("No active observations for update step") from e
 
-    responses = observations_and_responses.responses()
-    observations = observations_and_responses.observations()
-    errors = observations_and_responses.errors()
-    observation_keys = observations_and_responses.observation_keys()
-    indexes = observations_and_responses.index()
+    responses = observations_and_responses[
```

Comment:
If we return `observations_and_response` with an index set, we know that the first two columns are `observations` and `errors` and could get the rest by:

```python
observations_and_responses.iloc[:, 2:]
```